[ACA] [185409] Networking refactor (#125918)
Azure/azure-content: articles/spring-apps/migration/migrate-to-azure-container-apps-network.md

- A dedicated subnet for the Azure Container Apps service runtime subnet isn't required.
- Azure Container Apps provide a more flexible way to support UDR. In Azure Container Apps, there's no need to explicitly set the option `--outbound-type` to `userDefinedRouting` when provisioning Azure Spring Apps.

> For more information, see [User defined routes](../../container-apps/user-defined-routes.md) and [Control outbound traffic in Azure Container Apps with user defined routes](../../container-apps/user-defined-routes.md).

Reviewer: ```suggestion
For more information, see [Control outbound traffic in Azure Container Apps with user defined routes](../../container-apps/user-defined-routes.md).
```

Both Azure Spring Apps and Azure Container Apps support the use of custom DNS servers in a customer virtual network. We recommend adding Azure DNS IP `168.63.129.16` as the upstream DNS server in the custom DNS server.

> For more information, see the [DNS](../../container-apps/dns.md) section of [Networking in Azure Container Apps environment](../../container-apps/networking.md).

Reviewer: ```suggestion
For more information, see [DNS for virtual networks in Azure Container Apps environments](../../container-apps/dns.md).
```
